build/buildBuildpacksImages.sh

```diff
@@ -18,60 +18,25 @@ fi
 
 if [ -n "$BUILD_BUILDIMAGES_USING_NOCACHE" ]; then
 	echo
-	echo "Building buildpacks image(s) with NO cache..."
+	echo "Building buildpack runner image with NO cache..."
 	noCacheFlag='--no-cache'
 else
 	echo
-	echo "Building buildpacks image(s)..."
+	echo "Building buildpack runner image..."
 fi
 
-echo "-> Building stack base image: $ACR_PACK_STACK_BASE_IMAGE_REPO"
-echo
-cd "$BUILD_IMAGES_BUILD_CONTEXT_DIR"
-docker build -f "$PACK_STACK_BASE_IMAGE_DOCKERFILE" $noCacheFlag \
```

Comment:
Do we need to change things in the pipeline scripts to stop publishing these images?

Reply:
remove docker pull for pack-stack-base image in build/testIntegration.sh?